doc/src/examples/io/svg.cpp

```diff
@@ -1,14 +1,17 @@
 // Boost.Geometry (aka GGL, Generic Geometry Library)
 // QuickBook Example
 
-// Copyright (c) 2013 Barend Gehrels, Amsterdam, the Netherlands.
+// Copyright (c) 2013-2024 Barend Gehrels, Amsterdam, the Netherlands.
 
 // Use, modification and distribution is subject to the Boost Software License,
 // Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
 //[svg_mapper
-//` Shows the usage of svg_mapper
+//` Shows the usage of svg_mapper.
+//` Make sure to first call only the method [^add] and then call only the methods [^map] or [^text]. 
+//` Because [^add] calculates the bounding box internally and that should be finished
+//` before calling mapping functions. This might require two loops.
```

Comment:
Oops, this is also part of the other PR (I continued with that one). Anyway, no problem, will be harmonized

Reply:
✅